test/shopify-cli/api_test.rb

```diff
@@ -76,6 +76,23 @@ def test_query_fails_gracefully_with_internal_server_error
       end
     end
 
+    def test_query_fails_gracefully_with_internal_server_error_on_debug_mode
+      @context.stubs(:getenv).with('DEBUG').returns(true)
+      response = stub('response', code: '500', body: '{}')
+      HttpRequest.expects(:call).returns(response).times(4)
+      File.stubs(:read)
+        .with(File.join(ShopifyCli::ROOT, "lib/graphql/api/mutation.graphql"))
+        .returns(@mutation)
+
+      @context.expects(:debug).with(@context.message('core.api.error.internal_server_error_debug', "500\n{}"))
+      @context.expects(:puts).with(@context.message('core.api.error.internal_server_error'))
```

Comment:
I don't think we need to test this again, it's already covered by the previous test.

Reply:
That's true, but I figure it kind of works to check that both still get output. Slightly repetitive but more complete, I suppose.